libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -837,7 +824,7 @@ void AP_Baro::update(void)
         _alt_offset_active = _alt_offset;
     }
 
-    if (!_hil_mode) {
+    if (true) {
```

Comment:
seems not so useful ...

Reply:
This is simply to avoid changing a whole bunch of code which isn't touched by this PR.

I was planning on doing a follow-up patch which removed these.

Reply:
Removed it and reindented the block.